ethereum/forge-test/GasBenchmark.t.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "forge-std/Test.sol";
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
+import "./utils/WormholeTestUtils.t.sol";
+import "./utils/PythTestUtils.t.sol";
+
+contract GasBenchmark is Test, WormholeTestUtils, PythTestUtils {
+    // 19, current mainnet number of guardians, is used to have gas estimates
+    // close to our mainnet transactions.
+    uint8 constant NO_GUARDIANS = 19;
+    uint8 constant NO_SIGNERS = 13;
+
+    // We use 5 prices to form a batch of 5 prices, close to our mainnet transactions.
+    uint8 constant NO_PRICES = 5;
+
+    uint constant BENCHMARK_ITERATIONS = 50;
```

Comment:
Might be worth increasing this by two or three orders of magnitude, if it doesn't take too long. More iterations are always better 😄 

This would also help with the cold start problem.

Reply:
Good point. it is specially good for comparing the snapshots. Set it 1000. (5000 was out of gas)